generators/client/templates/angular/package.json.ejs

```diff
@@ -27,129 +27,129 @@
     "node_modules"
   ],
   "dependencies": {
-    "@angular/common": "7.2.12",
-    "@angular/compiler": "7.2.12",
-    "@angular/core": "7.2.12",
-    "@angular/forms": "7.2.12",
-    "@angular/platform-browser": "7.2.12",
-    "@angular/platform-browser-dynamic": "7.2.12",
-    "@angular/router": "7.2.12",
-    "@fortawesome/angular-fontawesome": "0.3.0",
-    "@fortawesome/fontawesome-svg-core": "1.2.17",
-    "@fortawesome/free-solid-svg-icons": "5.8.1",
-    "@ng-bootstrap/ng-bootstrap": "4.1.1",
+    "@angular/common": "8.0.0",
+    "@angular/compiler": "8.0.0",
+    "@angular/core": "8.0.0",
+    "@angular/forms": "8.0.0",
+    "@angular/platform-browser": "8.0.0",
+    "@angular/platform-browser-dynamic": "8.0.0",
+    "@angular/router": "8.0.0",
+    "@fortawesome/angular-fontawesome": "0.4.0",
+    "@fortawesome/fontawesome-svg-core": "1.2.18",
+    "@fortawesome/free-solid-svg-icons": "5.8.2",
+    "@ng-bootstrap/ng-bootstrap": "4.1.3",
```

Comment:
In `v4.2`, date objects are not being considered as valid (validations inbuilt into ng-bootstrap and introduced in `v4.2`). There is an open issue to track this: https://github.com/ng-bootstrap/ng-bootstrap/issues/3215

Reverting to version before that works fine.